charters/wot-wg-charter-draft-2019.html

```diff
@@ -178,39 +186,369 @@ <h2>Scope</h2>
+          The <a href="https://www.w3.org/TR/wot-architecture">WoT Architecture</a>
+          specification includes the definitions of several terms and concepts used to
+          define WoT specifications.  These are often related closely to use cases
+          and architectural patterns.  As WoT implementations mature, it will be 
+          necessary to define new use case, architectural patterns, and associated 
+          concepts and vocabulary.  In addition, descriptions of existing use cases
+          architectural patterns may need to be clarified or refactored to include
+          these new concepts.
+        </p>
+	<p>
+          Candidates for architectural vocabulary extension include a Producer concept 
+          and definition of information lifecycle and code deployment models.
+          The information lifecycle is especially relevant to the protection
+          of privacy.
+        </p>
+
+	<h3>TD Vocabulary</h3>
+	<p>
+          The
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>
+          context includes a core set of vocabulary.
+          This vocabulary can be extended by including additional context files.
+          However, to support new use cases and protocols in a consistent (interoperable) way,
+          especially in cases where JSON-LD processing is not possible,
+          extensions to this core vocabulary may be necessary.
+	</p>
+	<p>
+          Candidates for TD vocabulary extension include 
+          support for Data Schemas consistent with current versions of JSON Schema, 
+          extensions to Data Schemas to support XML and CBOR 
+          (and possibly other structured payload data formats), 
+          support for additional standard metadata such as location or device manufacturer,
+          and better support for default values.
+        </p>
+
+        <h3>Protocol Vocabulary</h3>
+	<p>
+          The
+          <a href="http://www.w3.org/TR/wot-thing-description">WoT Thing Description (TD)</a>
+          specification is augmented by a 
+          <a href="http://www.w3.org/TR/wot-protocol-binding">WoT Protocol Binding</a>
+          document which defines templates for how to use a TD to describe Things
+          using specific concrete protocols.
+          However, individual concrete protocols may require additional vocabulary
+          to define header options and other metadata.
+          These are defined using vocabulary extensions specific to each protocol.
+          Currently only a limited set of protocols are supported in a standard way.  
+          This work item will add support for an additional set of target protocols, 
+          where appropriate in collaboration with the defining standards body.
+        </p>
+        <p>
+          Candidates for extended protocol vocabulary include support 
+          for CoAP(S), CoAP(S)+WS(S), MQTT, and OPC-UA.
+        </p>
+
+        <h3>Smart Home Profile</h3>
```

Comment:
We could call this "Interoperability Profiles"

Reply:
I was just going to call them "Profiles" to keep the verbiage down.

Reply:
I decided to use "Interoperability Profiles" after all, at least in titles; it's a bit more self-explanatory.  Internal to the docs though I think we should try to use "Profiles" to keep the verbiage down.